orsopy/fileio/tests/test_orso.py

```diff
@@ -210,8 +212,8 @@ def test_user_data(self):
 
         data = np.zeros((100, 3))
         data[:] = np.arange(100.0)[:, None]
-        dct = {"ci": "1", "foo": ["bar", 1, 2, 3.5]}
-        info.user_data = dct
+        info.ci = 1
+        info.foo = ["bar", 1, 2, 3.4]
```

Comment:
What is `ls[0].info.user_data` explicitly supposed to be after the load on L220? Can you add a test for that?

Reply:
It used to be all non-orso attributes added to the first level. On all other levels it was just included in the dictrionary.

Now with the change of "from_dict" we can't create a class from a dict with unkown key anyway, so the solution for the user_data used at lower levels is the same as in the first level. (Create the class with orso attributes, then add your own attributes to the class. OR Create the class with `from_class(**dict)` and include the user attributes there.

The `Header.user_data` is still a read-only property that can be used to get the associated attributes. Do you think we should allow writing to user_data and then setting those attributes with the supplied dictionary?

Reply:
So now the `info.user_data` property should list `ci` and `foo` as keys in a dict, i.e. it's a pure accessor for retrieving non-standard items? But `ci` and `foo` are also accessible as attributes of `info`?

Reply:
Exactly. And `ci` and `foo` will be written and read to/from yaml.